.ci/jobs/e2e-testing-k8s-autodiscovery-daily-mbp.yml

```
@@ -0,0 +1,47 @@
+---
+- job:
+    name: e2e-tests/e2e-testing-k8s-autodiscovery-daily-mbp
+    display-name: End-2-End tests for Observability K8S Autodiscovery Pipeline
+    description: Run E2E K8S Autodiscovery test suite daily, including maintenance branches
+    view: E2E
+    project-type: multibranch
+    script-path: .ci/e2eTestingK8SAutodiscoveryDaily.groovy
+    scm:
+      - github:
+          branch-discovery: no-pr
+          head-filter-regex: '(master|7\.x|7\.13\.x|7\.12\.x)'
```

Including the upcoming 7.13.x. If we should wait for 7.14 artifacts to appear, let's add this commit:
```suggestion
          head-filter-regex: '(master|7\.x|7\.12\.x|7\.11\.x)'
```